src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const people = [...document.querySelectorAll('li')];
+
+function sortBySalary(arr) {
+  const salaries = arr.map((person) => {
+    const salary = person.getAttribute('data-salary');
+
+    return +salary.replace(/[$,]/g, '');
+  });
+
+  return salaries.sort((person1, person2) => person2 - person1);
```

Comment:
The comparison function in `salaries.sort()` should be applied to the original array of people, not the extracted salaries. Consider sorting `arr` directly using `arr.sort((person1, person2) => parseFloat(person2.getAttribute('data-salary').replace(/[$,]/g, '')) - parseFloat(person1.getAttribute('data-salary').replace(/[$,]/g, '')))`.